quizApp.js

```diff
@@ -56,17 +55,16 @@ function startQuiz() {
 
       const playerBtn = document.createElement("button");
       playerBtn.innerText = `${player} : 0`;
-      playerBtn.classList.add(`scorebtn-${index}`);
       playerContainer.appendChild(playerBtn);
       playerButtons.push(playerBtn);
 
       const correctIcon = document.createElement("i");
-      correctIcon.className = `fa fa-check-circle scorebtn-${index}`;
+      correctIcon.className = `fa fa-check-circle`;
       correctIcon.id = `icon-correct${index}`;
       playerContainer.appendChild(correctIcon);
 
       const wrongIcon = document.createElement("i");
-      wrongIcon.className = `fa fa-times-circle scorebtn-${index}`;
+      wrongIcon.className = `fa fa-times-circle`;
```

Comment:
You can just use a single quote notation (') here. Can you guess why? :)

Reply:
Yes, I will not add any values here. so i need not use template literal so i can use single quote or double quote.